### PR TITLE
ZIO: Enable snapshot releases

### PR DESCRIPTION
**Pull request by softinio**
related: https://github.com/zio/zio/issues/2724

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -12,9 +12,8 @@ addSbtPlugin("de.heikoseeberger"                 % "sbt-header"
 addSbtPlugin("ch.epfl.lamp"                      % "sbt-dotty"                     % "0.4.0")
 addSbtPlugin("org.scalameta"                     % "sbt-mdoc"                      % "1.3.6")
 addSbtPlugin("ch.epfl.scala"                     % "sbt-bloop"                     % "1.3.5")
-addSbtPlugin("com.dwijnand"                      % "sbt-dynver"                    % "4.0.0")
-addSbtPlugin("com.jsuereth"                      % "sbt-pgp"                       % "1.1.2")
-addSbtPlugin("org.xerial.sbt"                    % "sbt-sonatype"                  % "3.8.1")
 addSbtPlugin("org.scala-native"                  % "sbt-scala-native"              % "0.4.0-M2")
 addSbtPlugin("org.portable-scala"                % "sbt-scala-native-crossproject" % "0.6.1")
 addSbtPlugin("ch.epfl.scala"                     % "sbt-scalafix"                  % "0.9.11")
+addSbtPlugin("com.geirsson"                      % "sbt-ci-release"                % "1.5.0")
```
Comment (iravid) [inline project/plugins.sbt]: 1.5.2?

Comment (softinio) [reply project/plugins.sbt]: @iravid I don't see it on Maven yet: <https://search.maven.org/artifact/com.geirsson/sbt-ci-release>

Have a follow up PR to this to update README will upgrade it and remove `clean` from circleci release step as `1.5.2` once I see it on Maven. 

Prefer to stick to `1.5.0` for this PR as that is whats tested.

Comment (softinio) [reply project/plugins.sbt]: related: https://github.com/olafurpg/sbt-ci-release/issues/98

Comment (iravid) [reply project/plugins.sbt]: 👍

**Pull request merged**